README.md

```diff
@@ -59,6 +59,7 @@ You can pass the following options to `cmake ..`.
 * `-DALIGNED_MEMORY_MODEL` Build for aligned memory model only devices. Default is OFF.
 * `-DLOCAL_OPENSSL_BUILD` Whether or not to use local OpenSSL build. Default is OFF.
 * `-DCONSTRAINED_DEVICE` -- Change thread stack size to 0.5Mb, needed for Alpine.
+* `-DKVS_DEFAULT_STACK_SIZE` -- Change thread stack size to given value, overrides CONSTRAINED_DEVICE flag
```

Comment:
Would recommend specifying the units (or linking to the other readme where the units are specified).